charts/shoot-core/components/charts/network-policies/templates/allow-from-to-nginx.yaml

```diff
@@ -0,0 +1,25 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  annotations:
+    gardener.cloud/description: |
+      Allows Egress and Ingress from the nginx-controller to all traffice
+  name: gardener.cloud--allow-to-from-nginx
+  namespace: kube-system
+  labels:
+    origin: gardener
+spec:
+  podSelector:
+    matchLabels:
+      app: nginx-ingress
+  policyTypes:
+  - Ingress
+  - Egress
+  egress:
+  - to:
```

Comment:
I would also add all `Pods` to all namepaces to the egress as well.

Reply:
Actaully `- to: {}` should be sufficient. https://kubernetes.io/docs/concepts/services-networking/network-policies/#default-allow-all-ingress-traffic